src/storage/src/monitor/local_metrics.rs

```diff
@@ -29,6 +29,7 @@ pub struct StoreLocalStatistic {
     pub bloom_filter_true_negative_count: u64,
     pub bloom_filter_might_positive_count: u64,
     pub remote_io_time: Arc<AtomicU64>,
+    pub check_bloom_filter_counts: u64,
```

Comment:
We don't need to add this because it is essentially `bloom_filter_true_negative_count` + `bloom_filter_might_positive_count`.

Reply:
If you care more about the total check count, you can remove `bloom_filter_might_positive_count` and add `bloom_filter_check_count`.